package.json

```diff
@@ -26,6 +26,7 @@
     "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
+    "@ember/render-modifiers": "^2.0.4",
```

Comment:
Run-time code requires `{{did-insert}}` modifier. Therefore it should be a dependency instead of dev dependency. Fixed as part of the upgrade.